javascript-local-storage/data.js

```diff
@@ -0,0 +1,15 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+if (previousTodosJSON !== null) {
+  todos = JSON.parse(previousTodosJSON);
+}
+
+function unloadWindow(event) {
+  event.preventDefault();
```

Comment:
You don't want to do this, I think it will prevent the user from closing the page 😂